containerz/containerz.proto

```diff
@@ -257,6 +343,50 @@ message StartContainerRequest {
   // List of volumes that should be attached to the container.
   repeated Volume volumes = 7;
 
+  // The network to attach this container to. This could be 'host', 'bridged',
+  // or any other network available in the runtime.
+  string network = 8;
+
+  message Capabilities {
+    repeated string add = 1;
+    repeated string remove = 2;
```

Comment:
What are the semantics of `add` and `remove` here? What are we adding and removing from? Is it possible to just be declarative as to what we want the end state to be upon starting this container?

Reply:
By default a container starts with the following capabilities:
AUDIT_WRITE, CHOWN, DAC_OVERRIDE, FOWNER, FSETID, KILL, MKNOD, NET_BIND_SERVICE, NET_RAW, SETFCAP, SETGID, SETPCAP, SETUID, SYS_CHROOT.

Adding capabilities adds to this list while removing removes capabilities. Most applications will be fine with the default set and therefore this field can mostly be empty.

Reply:
Can we add a comment that says that this is the default set?

Reply:
per discussion -- clarify what happens if we do `add = CAP_SYSADMIN`, `remove = CAP_SYSTEM`.